web-client/public/js/graph.js

```diff
@@ -481,7 +481,7 @@ export var drawGraph = function (network) {
             var xOffsets;
             var color;
 
-            if (Math.abs(d.value / maxWeight) <= grayThreshold) {
+            if (normalize(d).toPrecision(4) <= grayThreshold) {
```

Comment:
Does _every_ call to `normalize` now follow with `toPrecision(4)`? Because if so, then we can integrate the `toPrecision(4)` conversion itself into the `normalize` function, thus leaving all of these lines as just `normalize(d)` once more, with `toPrecision(4)` already built-in.